src/Enterspeed.Cli/Services/EnterspeedClient/EnterspeedClient.cs

```diff
@@ -85,6 +86,8 @@ public async Task<RestResponse> ExecuteAsync(RestRequest request, CancellationTo
 
     private void AddHeaders(RestRequest request)
     {
+        request.AddHeader("X-Enterspeed-System-Information", $"Enterspeed-cli-{GetVersionNumber()}");
```

Comment:
Management frontend adds the header `X-Enterspeed-System=management-app/{version}`, so I think the header should be something like `X-Enterspeed-System=cli/{version}`

Reply:
I agree, it should be fixed now.